src/views/SettingsView.vue

```diff
@@ -55,6 +55,20 @@ const createOrUpdateUser = () => {
 	else createUser(userInfo as UserCreate)
 	editPanel.value?.togglePanel()
 }
+
+const sortPermissions = (permissions: AuthPermission[]) => {
+	return permissions.sort((one, two) => (one > two ? -1 : 1))
```

Comment:
In my opinion it is not necessary to sort them, I think this can be removed.

Reply:
ok, I'll remove that sort